src/components/Profile/ProfileInfoCard/editProfileDetailsModal.js

```diff
@@ -39,22 +47,43 @@ const EditProfileDetailsModal = ({ profileData, modelCloseCallback }) => {
   const [countryValidateError, setCountryValidateError] = useState(false);
   const [website, setWebsite] = useState(getData(profileData.website));
   const [websiteValidateError, setWebsiteValidateError] = useState(false);
-  const [websiteValidateErrorMessage, setWebsiteValidateErrorMessage] = useState("");
-  const [description, setDescription] = useState(getData(profileData.description));
-  const [descriptionValidateError, setDescriptionValidateError] = useState(false);
-  const [descriptionValidateErrorMessage, setDescriptionValidateErrorMessage] = useState("");
+  const [
+    websiteValidateErrorMessage,
+    setWebsiteValidateErrorMessage,
+  ] = useState("");
+  const [description, setDescription] = useState(
+    getData(profileData.description)
+  );
+  const [descriptionValidateError, setDescriptionValidateError] = useState(
+    false
+  );
+  const [
+    descriptionValidateErrorMessage,
+    setDescriptionValidateErrorMessage,
+  ] = useState("");
```

Comment:
Could you please revert this back to the original. Better to have these in a single line